File: Assets/VirtualCity/ProtoDefine/ReqGetOneDevInfoMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class ReqGetOneDevInfoMessage {
    /**玩家id*/
[ProtoMember(1)]
    public long? friendId;
    /**建筑id*/
[ProtoMember(2)]
    public string devId;

    public ReqGetOneDevInfoMessage() {
    }

    public ReqGetOneDevInfoMessage(long? friendId, string devId) {
        this.friendId = friendId;
        this.devId = devId;
    }

    public long? getFriendId() {
        return friendId;
    }

    public void setFriendId(long? friendId) {
        this.friendId = friendId;
    }

    public string getDevId() {
        return devId;
    }

    public void setDevId(string devId) {
        this.devId = devId;
    }
}
}

File: Assets/VirtualCity/ProtoDefine/Java/building/ReqGetOneDevInfoMessage.java
package com.kingston.jforgame.server.game.building.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.building.BuildingDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.BULIDING,cmd = BuildingDataPool.REQ_GETONEDEVINFO)
public class ReqGetOneDevInfoMessage extends Message {
    /**玩家id*/
    @Protobuf(order = 1)
    private Long friendId;
    /**建筑id*/
    @Protobuf(order = 2)
    private String devId;

    public ReqGetOneDevInfoMessage() {
    }

    public ReqGetOneDevInfoMessage(Long friendId, String devId) {
        this.friendId = friendId;
        this.devId = devId;
    }

    public Long getFriendId() {
        return friendId;
    }

    public void setFriendId(Long friendId) {
        this.friendId = friendId;
    }

    public String getDevId() {
        return devId;
    }

    public void setDevId(String devId) {
        this.devId = devId;
    }
}
